### PR TITLE
cifs-utils and talloc

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: cifs-utils.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - libcap-dev
+      - keyutils-dev
+      - krb5-dev
+      - autoconf
+      - automake
+      # - samba-dev
```
Comment (imjasonh) [inline cifs-utils.yaml:21]: Why is this commented out?

File: cifs-utils.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      --prefix=/usr \
+      --disable-systemd \
+      --sbindir=/usr/bin
+      make
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      make DESTDIR="${{targets.destdir}}" ROOTSBINDIR=/usr/bin install
+      # set mount.cifs uid, to enable none root mounting form fstab
+       chmod +s "${{targets.destdir}}"/usr/bin/mount.cifs
```
Comment (imjasonh) [inline cifs-utils.yaml:51]:
```suggestion
      chmod +s "${{targets.destdir}}"/usr/bin/mount.cifs
```

File: cifs-utils.yaml (hunk 1 above)
Comment (developer-guy) [reply cifs-utils.yaml:21]: I looked at the packages specified in the [APKBUILD](https://git.alpinelinux.org/aports/tree/main/cifs-utils/APKBUILD#n10) the `samba-dev` is one of them but it seems quite complicated so I tried to build this package without it to see whether we depend on `samba-dev`. At a result, it seems we don't. I also checked [here](https://gitlab.archlinux.org/archlinux/packaging/packages/cifs-utils/-/blob/main/PKGBUILD?ref_type=heads), there is no `samba-dev`. PTAL @kaniini

Comment (kaniini) [reply cifs-utils.yaml:21]: We definitely need `samba` for `cifs-utils` to work in a meaningful way.

**Comment by xnox** [comment]
Superseded by https://github.com/wolfi-dev/os/pull/11504/files

**Pull request closed without merging**